tests/test_butler.py

```diff
@@ -551,6 +551,11 @@ def testConstructor(self):
                     uri = Butler.get_repo_uri("label")
                     butler = Butler(uri, writeable=False)
                     self.assertIsInstance(butler, Butler)
+                    butler = Butler("label", writeable=False)
+                    self.assertIsInstance(butler, Butler)
+                    with self.assertRaises(FileNotFoundError) as cm:
+                        Butler("not_there", writeable=False)
+                    self.assertIn("label", str(cm.exception))
```

Comment:
Can do `assertRaisesRegex` to match exception message, same for the change below.